src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -115,13 +121,16 @@ export const WorkspaceForm = ({
       const apps = category2Applications[currentKey];
       const features = apps
         .filter(
-          ({ navLinkStatus, chromeless }) =>
-            navLinkStatus !== AppNavLinkStatus.hidden && !chromeless
+          ({ navLinkStatus, chromeless, featureGroup }) =>
+            navLinkStatus !== AppNavLinkStatus.hidden &&
+            !chromeless &&
+            !featureGroup?.includes('ADMIN')
```

Comment:
Should it check `featureGroup?.includes('WORKSPACE')`? because one feature could be 'WORKSPACE' and 'ADMIN' at the same time. Also one question, if a feature didn't have `featureGroup` specified, do we consider it as workspace feature?

Reply:
For now, the application doesn't include `featureGroup` field will be treated as a workspace feature. Do we need to remove them from the workspace feature list?

Reply:
> For now, the application doesn't include `featureGroup` field will be treated as a workspace feature. Do we need to remove them from the workspace feature list?

Makes sense to me, then we can check if `featureGroup` is empty or contains `WORKSPACE`